versioned_docs/version-v2.18.x/extend/mvd-conda.md

```diff
@@ -0,0 +1,192 @@
+# Using Conda to make and manage packages of Application Framework Plug-ins
```

Comment:
@1000TurquoisePogs - This file was added to the v2.18 sidebar. Should it also be added to the sidebars for the Zowe V3 releases (3.0 and 3.1)?

Reply:
It seems that there was some reorganization of the Zowe Desktop content.
Somehow, this file is in the live site under `extend-desktop`.
https://docs.zowe.org/v2.18.x/extend/extend-desktop/mvd-conda 

I second Ana's comment 🙏 
@1000TurquoisePogs, it'd be good to have your thoughts as to where it should go!

Reply:
This file was removed in v3 as part of cleanup - it was something that never caught on so I didn't want people to be enticed by it going forward.

Reply:
Is it possible this file was intended to be removed in v2 as well?
I don't know of anyone outside of the Rocket developers who ever touched the content mentioned on this page.